content/code-signing/ios-code-signing.md

```diff
@@ -112,3 +112,7 @@ To export the provisioning profile:
 2. Navigate to **Certificates, Identifiers & Profiles > Profiles**.
 3. Select the provisioning profile you would like to export and click **Download**.
 4. Save it to have it ready.
+
+## Example
+
+If you are yearning for a more detailed example, Souvik Biswas has written a complete step-by-step <a href="https://blog.codemagic.io/distributing-native-ios-sdk-with-flutter-module-using-codemagic/" target="_blank" onclick="sendGtag('Link_in_docs_clicked','distributing-native-ios-sdk-with-flutter-module-using-codemagic')">overview about distributing a Native iOS project,</a> including both code signing and publishing it to App Store Connect.
```

Comment:
Let's not mention author names in docs, we don't want it to read too marketingy. Also, I'm not sure if iOS code signing doc is the right place for native iOS example. That particular reference would be more appropriate at the end of the yaml file doc or in the yaml template we will have about native iOS + Flutter module.

Reply:
Actually, we do have other blog posts that cover only the iOS code signing part, not sure how up to date they are, though.